CHANGELOG.md

```diff
@@ -105,6 +105,7 @@ Thankyou! -->
     1. Added `control_parameters` and `status_details` to the compliance object. #1219
     1. Added `geodetic_altitude`, `height`, `horizontal_accuracy`, and `pressure_altitude` to `location`. #1169
     1. Added `location` to `managed_entity`. #1169
+    1. Added `kernel_ver` to `os` object Linux extension.
```

Comment:
Under "Dictionary Attributes," need to add a line for "Added kernel_ver as a string_t"